aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/contexts/LambdaSegmentContext.java

```diff
@@ -80,6 +80,7 @@ public Subsegment beginSubsegment(AWSXRayRecorder recorder, String name) {
                     ? new SubsegmentImpl(recorder, name, parentSegment)
                     : Subsegment.noOp(parentSegment, recorder);
             subsegment.setParent(parentSegment);
+            subsegment.putAllAws(recorder.getAwsRuntimeContext());
```

Comment:
If user creates multiple "root subsegments", looks like each will have a copy of this information.
Do we want only one of the subsegments to have this information?
I don't know which is more reasonable: one or all subsegments with this data.

Reply:
We are adding this information to all subsegments and not just "root subsegments" (not sure @jj22ee what you mean by this), right? 
So it seem redundant to add same information to all the subsegments. Can we possibly add this to the segment created by Lambda runtime?